### ISSUE TITLE
Simulate starting a navigation

### ISSUE THREAD

**Issue by stefanieschreiner** [amosproj/amos2022ws02-automotive-test-app#8]
## User story
1. As a User
2. I want to simulate the start of a navigation by pressing a button
3. So that it is possible to test a new navigation app.

It should be possible to start a navigation immediately, delayed by 30 secs or delayed by 1min.

## Acceptance criteria
* In the navigation context there is a possibility to simulate the start of a navigation immediately.
* In the navigation context there is a possibility to simulate the start of navigation after 30 seconds or after one minute.
* The delayed start of the navigation must also be done when the app is running in the background.

## Notes
The necessary AOSP service is the CarAppFocusManager: https://developer.android.com/reference/android/car/CarAppFocusManager
The start of a navigation is simulated by requesting the focus at the CarAppFocusManager.
A listener can be registered at the CarAppFocusManager, which receives information about the current status of the app focus (addFocusListener()).

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
